plugins/repoclosure.py

```diff
@@ -75,6 +75,19 @@ def _get_unresolved(self, arch=None):
         if arch is not None:
             available = available.filter(arch=arch)
         pkgs = set()
+
+        # available seems to contain latest-per-repo but we want the latest
```

Comment:
Somehow I didn't get the advantage of the presented approach. Somehow it sound to me like improvement that solve only particular case but it has limited impact in general. Please can you provide user-case, that can explain the reason of pull-request. Important point of view is also the behavior of yum in comparison with dnf and with presented pull-request.